### PR TITLE
[Feature] Upgrade Package Support To Laravel 8

### PR DESCRIPTION
**Pull request by dljfield**
This moves us to L8. Most of the work has been put into making the tests pass, as we have moved to using the new, "improved" factories, rather than add the legacy factories package.

Additionally we have moved models to a `Models` namespace, to be in line with how a new L8 project is structured in that regard.

### CODE REVIEW

File: phpunit.xml.bak [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mauricius) [inline phpunit.xml.bak:1]: You can delete this one as well

File: tests/TestCase.php [hunk 2]
```diff
@@ -19,19 +23,19 @@ protected function setUp(): void
...
-     * @param \Illuminate\Foundation\Application $app
-     * @return array
-     */
-    protected function getPackageAliases($app)
-    {
-        return [
-            'Form' => FormFacade::class,
-        ];
-    }
+//    /**
```
Comment (mauricius) [inline tests/TestCase.php:31]: I think you can remove also this code. We're not using the HtmlCollective package anymore.

**Review by mauricius — APPROVED** [review]

File: phpunit.xml.bak (hunk 1 above)
Comment (dljfield) [reply phpunit.xml.bak]: Baleted

File: tests/TestCase.php (hunk 2 above)
Comment (dljfield) [reply tests/TestCase.php]: Removed the commented out code

**Pull request merged**